.github/workflows/ruby.yml

```
@@ -7,51 +7,36 @@ jobs:
     name: Ruby specs
     runs-on: ${{ matrix.os }}
     continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' || matrix.experimental }}
-    env:
-      BUNDLE_JOBS: 4
-      BUNDLE_RETRY: 3
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
         node: [14]
-        ruby: [
-          2.7,
-          3.0
-        ]
-        gemfile: [
-          "gemfiles/Gemfile-rails.5.2.x",
-          "gemfiles/Gemfile-rails.6.0.x",
-          "gemfiles/Gemfile-rails.6.1.x"
-        ]
+        ruby:
+          - 2.7
+          - "3.0"
+        gemfile:
+          - gemfiles/Gemfile-rails.5.2.x
+          - gemfiles/Gemfile-rails.6.0.x
+          - gemfiles/Gemfile-rails.6.1.x
         exclude:
-          - ruby: 2.4
-            gemfile: gemfiles/Gemfile-rails.6.0.x
-          - ruby: 2.4
-            gemfile: gemfiles/Gemfile-rails.6.1.x
           - ruby: 2.5
             gemfile: gemfiles/Gemfile-rails.6.1.x
-          - ruby: 3.0
+          - ruby: "3.0"
             gemfile: gemfiles/Gemfile-rails.5.2.x
         experimental: [false]
         include:
           - ruby: 2.7
             os: ubuntu-latest
             gemfile: gemfiles/Gemfile-rails-edge
             experimental: true
-          - ruby: 3.0
+          - ruby: "3.0"
             os: ubuntu-latest
             gemfile: gemfiles/Gemfile-rails-edge
             experimental: true
 
     steps:
```

`BUNDLE_GEMFILE` needs to be set:
```suggestion
    env:
      BUNDLE_GEMFILE: ${{ matrix.gemfile }}
    steps:
```

---

Perfect, that was it!

---

That fixed my cut-and-paste mistake, thanks!